src/order/upgrade/privateCloud/order-upgrade-privateCloud.service.js

```diff
@@ -0,0 +1,11 @@
+angular
+    .module("ovh-api-services")
+    .service("OvhApiOrderUpgradePrivateCloud", function ($injector) {
+
+        "use strict";
+        return {
+            V6: function () {
```

Comment:
```suggestion
            v6: function () {
```
to keep consistency with other services ?

Reply:
Copy pasted from a call with a `V6` name, I don't mind changing though